usagov/usagov-2021
# USAGOV-1983-New-Menu-Color

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1983

## Description

Changed the vertical bar of current page back to #D83933

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Go to any second level page (i.e http://localhost/immigration-case-status or http://localhost/government-grants-and-loans) 
- Make sure that the left vertical bar on the menu is #D83933 but the text is #AE1A0A
![Screenshot 2024-10-10 at 2 38 43 PM](https://github.com/user-attachments/assets/ad328e3b-7d2f-4a4c-badb-f795641549da)

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
